src/utils/execa.js

```diff
@@ -1,12 +1,13 @@
-const { env } = require('process')
+import { env } from 'process'
 
-const execaLib = require('execa')
+import execaLib from 'execa'
 
 // This is a thin layer on top of `execa` that allows consumers to provide an
 // alternative path to the module location, making it easier to mock its logic
 // in tests (see `tests/utils/mock-execa.js`).
 
 // eslint-disable-next-line import/no-dynamic-require
+// TODO: does not work either!
 const execa = env.NETLIFY_CLI_EXECA_PATH ? require(env.NETLIFY_CLI_EXECA_PATH) : execaLib
```

Comment:
@ehmicky do you have a better idea as turning that into an async function? Do we need that still?

Reply:
`NETLIFY_CLI_EXECA_PATH` feels a little hacky to me, and seems to be only needed for testing purpose, so if there is a way to remove it, we should try that.